File: javascript/problem1.js
console.log("#######################");
console.log("Project Euler Problem 1");
console.log("#######################\n");

let sum = 0;

for (let i = 1; i < 1000; i++) {
  //See if number is divisible by 3 or 5
  if (i % 3 === 0 || i % 5 === 0) {
    //if number is divisible by 3 or 5, add to the sum
    sum = sum + i;
  }
}

console.log("Problem 1 Answer is " + sum);

File: python/problem1.py
#!/opt/homebrew/bin/python3

print("\n#######################")
print("Project Euler Problem 1")
print("#######################\n")

sum = 0

for i in range(1000):
    if i % 3 == 0 or i % 5 == 0:
        sum += i

print("Problem 1 answer is ", sum, "\n")
